lib/internal/main/run_main_module.js

```diff
@@ -8,6 +10,8 @@ prepareMainThreadExecution(true);
 
 markBootstrapComplete();
 
+RegExpPrototypeExec(/^/, '');
```

Comment:
This could use a comment to explain its existence, otherwise someone might come along and try to remove it as it looks seemingly useless.